simple/stats/schema_constants.py

```diff
@@ -19,21 +19,23 @@
 SV_HIERARCHY_PROPS_BLOCKLIST: set[str] = {
     "dcid", "typeOf", "isPublic", "provenance", "resMCFFile", "keyString",
```

Comment:
i wonder if you want to sort set items and show one item per line (if lint permits), so future diff is easier to see.